src/Symfony/Bridge/Twig/Mime/WrappedTemplatedEmail.php

```diff
@@ -93,15 +90,15 @@ public function getReturnPath(): string
     /**
      * @return $this
      */
-    public function addFrom(string $address, string $name = null): self
+    public function addFrom(string $address, string $name = ''): self
```

Comment:
this should be documented in the upgrade guide too, as the parameter type changed from `string|null` to `string`, which is also a BC break.

Reply:
The parameter type was `string` and `null` was only to express that no arguments was passed. Anyway, I think that's minor and we should not waste our time on such things.